independent-publisher-connectors/PaySpace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4476 @@
+        "description": "Retrieves a list of active employee biographical details as of the today's date and based on the specified company-id.",
+        "operationId": "GetACollectionOfEmployees",
+        "x-ms-visibility": "advanced",
+        "parameters": [
+          {
+            "name": "company-id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "{{company-id}} The Company Id",
+            "x-ms-summary": "{{company-id}}"
+          },
+          {
+            "name": "$orderby",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "(string) - Specifies the order in which items are returned",
+            "x-ms-summary": "Optional by {{$employee-field}}"
+          },
+          {
+            "name": "$top",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": " (integer($int32)) - Limits the number of items returned from a collection",
+            "x-ms-summary": "Optional $top"
+          },
+          {
+            "name": "$skip",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": " (integer($int32)) - Excludes the specified number of items of the queried collection from the result",
+            "x-ms-summary": "Optional $skip"
+          },
+          {
+            "name": "$count",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": " (true/false) - Return total count in response",
+            "x-ms-summary": "Optional $count"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": " Optional - Filter by field eg. \"EmployeeNumber eq 'Emp01'\"",
+            "x-ms-summary": "Optional $filter"
+          },
+          {
+            "name": "$select",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": " Optional (string) - Returns only the fields specified",
+            "x-ms-summary": "Optional $select"
+          },
+          {
+            "name": "CustomAuthHeader",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Bearer token value (without the keyword Bearer)",
+            "x-ms-summary": "Bearer token"
+          },
+          {
+            "name": "CustomEnvironmentHeader",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Variable that is either api (for PROD) or apistaging (for STAGING)",
+            "x-ms-summary": "Environment Variable"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
@mint-virgil Please change to a proper description. for example you can use "OK", "Success", "Bad Request", ...etc.
Please fix all responses' descriptions in your connector swagger file.

Reply:
resolved in previous commit